### DISCUSSION TITLE
Mapped Network Drive as target source

### DISCUSSION THREAD

**Discussion by Dave1342** [EricZimmerman/KapeFiles#788] [Q&A]
Hey guys,
I have several network drives mapped to my system, but Kape does not recognize them in the target source. Neither gkape nor kape (--tsource 'N:\' not available. Exiting).
Any idea? 
Thanks for your help.

**Comment by AndrewRathbun** [comment]
Which version of KAPE? Have you tried using UNC paths?

**Reply by Dave1342** [reply]
I was using V 1.2.0.0. In the meanwhile I updated to the latest Version 1.3.0.2, but still cannot find the network drive.
When I use UNC paths it seems to find it.

**Comment by EricZimmerman** [comment]
How did you map them? As an admin? If not it won't work. It's a windows thing.

**Reply by Dave1342** [reply]
OK, that was the issue...I didn`t connect as admin. Thx a lot. No it works fine.

**Comment by EricZimmerman** [comment]
I would not recommend searching across the UNC share unless you're only interested in finding active files or you do not want special files like the MFT volume shadow copy base files and so on

It is almost always the correct choice to run kape directly on the end point as opposed to running it against the UNC share

**Reply by Dave1342** [reply]
I will consider that. Thank you.
